packages/@jsii/benchmarks/index.ts

```diff
@@ -0,0 +1,53 @@
+import * as Benchmark from 'benchmark';
+import * as cp from 'child_process';
+import * as fs from 'fs-extra';
+import { Compiler } from 'jsii/lib/compiler';
+import { loadProjectInfo } from 'jsii/lib/project-info';
+import * as os from 'os';
+import * as path from 'path';
+import { promisify } from 'util';
+
+// Always run against the same version of CDK source
+const CDK_TAG = 'v2.21.1';
+const exec = promisify(cp.exec);
+async function setupAwsCdkLib(): Promise<{ awsCdkLib: string }> {
+  const repoDir = await fs.mkdtemp(path.join(os.tmpdir(), 'jsii-cdk-bench'));
```

Comment:
I would really, really like it if that temporary directory was cleaned up before exit.